src/Elliptic/Systems/Elasticity/Tags.hpp

```diff
@@ -0,0 +1,60 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+/*!
+ * \ingroup EllipticSystemsGroup
+ * \brief Items related to solving linear elasticity problems
+ *
+ * \details In linear elasticity problems we solve for the displacement vector
+ * field \f$\boldsymbol{u}\f$ in an elastic material that responds to an applied
+ * force \f$\boldsymbol{F}\f$. In this static approximation the equation of
+ * motion reduces to the elliptic equation \f[
+ * -\nabla_i T^{ij} = F^j
+ * \f]
+ * where the stress \f$T^{ij}\f$ is related to the strain
+ * \f$S_{ij}=\nabla_{(i}u_{j)}\f$ by a linear constitutive relation
+ * \f$T^{ij}=\chi^{ijkl}S_{kl}\f$ that describes the elastic properties of the
```

Comment:
I prefer \f$Y_{ijkl}\f$ for the Young's tensor

Reply:
sure, makes sense to be consistent with at least _one_ paper 😄 so let's make that yours